Samples/Tutorials/CMakeLists.txt

```diff
@@ -14,9 +14,14 @@ file(COPY ${OGRE_CONFIG_DIR}/plugins.cfg ${OGRE_CONFIG_DIR}/resources.cfg
      DESTINATION ${CMAKE_BINARY_DIR})
 ## [discover_ogre]
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -pedantic")
```

Comment:
were you trying to get that deprecation waring? this does not seem to be needed.

Reply:
In all code I am porting from tutorials. Just for you not to correct me all the time.

Reply:
you should rather set this in cmake-gui instead of hardcoding it here